prrget/src/App.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import './App.css';
+import Searchbar from "./Searchbar.js";
+
+const axios = require('axios');
+
+class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      cats: [],
+      catName: ''
+    };
+    this.getCat = this.getCat.bind(this);
+    this.onCatNameChange = this.onCatNameChange.bind(this);
+    this.getSearchedCat = this.getSearchedCat.bind(this);
+  }
+  componentDidMount () {
+    this.getCat();
+    console.log('component mounts')
+
+  }
+
+  getCat () {
+    console.log(this.state.catName)
+    axios.get(`http://192.168.1.70:5300/cats/${this.state.catName}`)
```

Comment:
Change the URL to start with /cats. This will ensure it works regardless of the machine IP